### DISCUSSION TITLE
Supabase + ExpressJS + OAuth tutorials?

### DISCUSSION THREAD

**Discussion by oleg-agapov** [supabase/supabase#2848] [Questions]
I'm looking for a way to integrate Supabase auth with Expressjs app, including Google oauth provider and user sessions.

There are a couple of examples with Nextjs/Svelte SSR, which claim to be using backend cookies, but I can't wrap my head around those examples.

Maybe you guys know some nice tutorials on the topic?

**Comment by void-pulsar** [comment]
Looking for this too!

**Comment by DhruvSathavara** [comment]
looking for the same 👀

**Comment by xrealblue** [comment]
same
